README.md

```diff
@@ -0,0 +1,69 @@
+# README 🎫
+
+Kensatsu is a tool that allows mapping IC card inputs to input events.
+
+## But why?
+
+When I was on my honeymoon in Japan, I noticed that pretty much every arcade machine had a transit card reader on it. It allows you to play arcade games without coins on hand, with your card being directly debited whatever the game costs to play. When I got an arcade game to play at home, I decided I wanted to replicate the setup myself by hooking up an IC card reader to insert virtual coins. If you want to do the same, Kensatsu is here to help.
+
+## How does it work?
+
+Kensatsu sits between a USB card reader and your game. It waits for your card reader to register that a transit card has been tapped, then emits a USB input event that other software on your system can read. Right now, the main intended use for Kensatsu is [OpenJVS](https://github.com/openjvs/openjvs), an open source implementation of the JVS standard used by real arcade boards. If you're using OpenJVS, you can hook up your card reader to the same Raspberry Pi you connect your controller to.
```

Comment:
> a USB input event

Is it a USB input event? I was under the impression it was something else

Reply:
Good point, it's just an input event. Fixed.